bedrock/firefox/templates/firefox/base.html

```diff
@@ -15,7 +15,7 @@
 {% block facebook_id %}14696440021{# facebook.com/Firefox #}{% endblock %}
 {% block twitter_id %}firefox{% endblock %}
 
-{% block site_css %}
+{% block page_css %}
```

Comment:
Probably my mistake when updating site_css to page_css. This is a base template so it should stay site_css. Fixing this.